app/src/main/res/layout/home_fragment.xml

```diff
@@ -20,24 +21,24 @@
             android:layout_marginTop="30dp"
             android:background="@drawable/frame"
             android:backgroundTint="#C6B2EA"
+            android:onClick="@{() -> viewModel.onD20Clicked()}"
             android:src="@drawable/d20_svg"
             app:layout_constraintStart_toStartOf="parent"
-            app:layout_constraintTop_toTopOf="parent"
-            android:onClick="@{() -> viewModel.onD20Clicked()}"/>
```

Comment:
Let's not use viewbinding. 
It doesn't play well with observable stuff like LiveData and it mixes view and logic. 

Instead just set a click listener in your fragmant code